api/overdrive.py

```diff
@@ -746,50 +749,23 @@ def json(self):
         return json.loads(self.content)
 
 
-class DummyOverdriveAPI(OverdriveAPI):
+class DummyOverdriveAPI(MockOverdriveAPI, OverdriveAPI):
```

Comment:
I don't have plans to do this. The mock API class in circulation and its superclass in core have the same relationship as the actual API class in circulation and its superclass in core. The circulation API does patron authentication to do stuff like borrow books, and the core API uses the library credentials to do stuff like check availability.

We could in theory merge the two classes but I think I've gotten rid of most of the confusion by removing the redundant code from the circulation mock API.